Sources/SwiftCurrent_SwiftUI/Views/NavigationWrapper.swift

```diff
@@ -0,0 +1,21 @@
+//  swiftlint:disable:this file_name
+//  NavigationWrapper.swift
+//  SwiftCurrent_SwiftUI
+//
+//  Created by Tyler Thompson on 8/27/21.
+//  Copyright © 2021 WWT and Tyler Thompson. All rights reserved.
+//
+
+import SwiftUI
+
+@available(iOS 14.0, macOS 11, tvOS 14.0, watchOS 7.0, *)
+extension View {
+    func navLink<D: View>(to destination: D, isActive: Binding<Bool>) -> some View {
```

Comment:
I left this in here, it's the workaround we had to use for nav links that are defaulted to true.